make run-local missing charts

When running `make run-local`, I get this error.

```
. ./scripts/install_local.sh local build/resources
namespace "local" created
error: the path "build/resources" does not exist
make: *** [run-local] Error 1
```

Shouldn't the manifests task be added to the run-local one?

--- a/Documentation/install/local-values-shift.yaml
+++ b/Documentation/install/local-values-shift.yaml
@@ -26,3 +26,7 @@ servicebroker:
   service:
     internalPort: 8080
     brokerPort: 8005
+
+catalog_sources:
+ - tectonic-ocs
+ - upstream-components

--- a/Documentation/install/local-values.yaml
+++ b/Documentation/install/local-values.yaml
@@ -2,6 +2,7 @@ rbacApiVersion: rbac.authorization.k8s.io
 namespace: local
 watchedNamespaces: local
 catalog_namespace: local
+
 alm:
   replicaCount: 1
   image:
@@ -26,3 +27,7 @@ servicebroker:
   service:
     internalPort: 8080
     brokerPort: 8005
+
+catalog_sources:
+ - tectonic-ocs
+ - upstream-components

--- a/Makefile
+++ b/Makefile
@@ -58,22 +58,21 @@ build/chart/templates/%.yaml: deploy/chart/templates/%.yaml
 	mkdir -p build/chart/templates
 	cp $< $@
 
-manifests: $(CHARTS) build/chart/Chart.yaml build/chart/values.yaml
+manifests: clean $(CHARTS) build/chart/Chart.yaml build/chart/values.yaml
 	mkdir -p build/resources
 	helm template -n olm -f build/chart/values.yaml build/chart --output-dir build/resources
 
 run-local: values_file = Documentation/install/local-values.yaml
 run-local: ver=0.0.0-local
-run-local: release
+run-local: manifests
 	. ./scripts/build_local.sh
-	. ./scripts/install_local.sh local build/resources
+	. ./scripts/install_local.sh local build/resources/olm/templates
 
 run-local-shift: values_file = Documentation/install/local-values-shift.yaml
 run-local-shift: ver=0.0.0-local
 run-local-shift: manifests
-	sed -i 's/rbac.authorization.k8s.io/authorization.openshift.io/' build/resources/02-alm-operator.rolebinding.yaml
 	. ./scripts/build_local_shift.sh
-	. ./scripts/install_local.sh local build/resources
+	. ./scripts/install_local.sh local build/resources/olm/templates
 
 e2e-local: values_file = test/e2e/e2e-values.yaml
 e2e-local: manifests
@@ -191,4 +190,5 @@ ansible-release: $(YQ)
 	# link newest release into playbook
 	ln -sfF ../../../../deploy/aos-olm/$(ver) deploy/aos-olm/playbook/private/roles/olm
 
+# make ver=0.3.0 release
 release: tectonic-release upstream-release ansible-release

